RestaurantReservation.Db/Customer.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace RestaurantReservation.Db
+{
+    public class Customer
+    {
+        [Key]
+        public int customer_id { get; set; }
+        public string first_name { get; set; } 
+        public string last_name { get; set;}
+        public string email { get; set; }
+        public string phone { get; set; }
+        public ICollection<Reservation> Reservations { get; set; }
```

Comment:
Why not to store reservations Ids instead of the whole object. If needed, we can fetch it from the database.

Same comment for all collections.